manager/src/main/java/com/redhat/service/smartevents/manager/dns/openshift/DnsServiceOpenshiftImpl.java

```diff
@@ -0,0 +1,99 @@
+package com.redhat.service.smartevents.manager.dns.openshift;
+
+import java.time.Duration;
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.amazonaws.services.route53.model.Change;
+import com.amazonaws.services.route53.model.ChangeAction;
+import com.amazonaws.services.route53.model.ChangeBatch;
+import com.amazonaws.services.route53.model.ChangeResourceRecordSetsRequest;
+import com.amazonaws.services.route53.model.InvalidChangeBatchException;
+import com.amazonaws.services.route53.model.RRType;
+import com.amazonaws.services.route53.model.ResourceRecord;
+import com.amazonaws.services.route53.model.ResourceRecordSet;
+import com.redhat.service.smartevents.manager.ShardService;
+import com.redhat.service.smartevents.manager.dns.DnsService;
+
+import io.quarkus.arc.properties.IfBuildProperty;
+import io.smallrye.mutiny.Uni;
+
+@ApplicationScoped
+@IfBuildProperty(name = "event-bridge.k8s.orchestrator", stringValue = "openshift", enableIfMissing = true)
+public class DnsServiceOpenshiftImpl implements DnsService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DnsServiceOpenshiftImpl.class);
+
+    private static final Duration DEFAULT_TIMEOUT = Duration.ofSeconds(5);
+
+    DnsConfigOpenshiftProvider dnsConfigOpenshiftProvider;
+
+    ShardService shardService;
+
+    @Inject
+    public DnsServiceOpenshiftImpl(ShardService shardService, DnsConfigOpenshiftProvider dnsConfigOpenshiftProvider) {
+        this.shardService = shardService;
+        this.dnsConfigOpenshiftProvider = dnsConfigOpenshiftProvider;
+    }
+
+    @Override
+    public String buildBridgeHost(String bridgeId) {
+        return bridgeId + dnsConfigOpenshiftProvider.getSubdomain();
+    }
+
+    /**
+     * Creates a new DNS record in the hosted zone with the "simple" router policy. For complex DNS resolution we have to change this accordingly.
+     *
+     * @param bridgeId
+     * @return
+     */
+    @Override
+    public Boolean createDnsRecord(String bridgeId) {
+        LOGGER.info(String.format("creating DNS zone for bridge '%s'", bridgeId));
```

Comment:
@r00ta You don't need the `String.format` inside the `LOGGER.info`, you can use placeholders in the log statement. I also think it could be useful to provide more details of the sub-domain we're creating in the logs here just to aid with debugging e.g:

```
LOGGER.info("Creating sub-domain "{}" for Bridge "{}" in Zone "{}", buildBridgeHost(bridgeId), bridgeId, dnsConfigOpenshiftProvider.getHostedZoneId());
```